src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -234,7 +234,7 @@ def acs_install_cli(cmd, client, resource_group, name, install_location=None, cl
 
 
 def _ssl_context():
-    if sys.version_info < (3, 4):
+    if sys.version_info < (3, 4) or (in_cloud_console() and platform.system() == 'Windows'):
         return ssl.SSLContext(ssl.PROTOCOL_TLSv1)
```

Comment:
Use ssl.PROTOCOL_TLS instead
this will select highest protocol version that both the client and server support and this setting is not deprecated.

https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLS

Reply:
Depending on the exact Python version, this may fail if you look at both notes.
However, the Windows MSI ships with Python 3.6 so it should be good.
For users who install the CLI through some other method and have Python 3.5, it may fail.

https://docs.python.org/2/library/ssl.html#ssl.PROTOCOL_TLS
https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLS

Reply:
That's much more future-proof and should be safe to use if I wrap it in a `try/except`.